spec-main/api-web-contents-spec.ts

```diff
@@ -1155,6 +1155,10 @@ describe('webContents module', () => {
             res.end();
           } else if (req.url === '/redirected') {
             res.end('<html><script>window.localStorage</script></html>');
+          } else if (req.url === '/first-window-open') {
+            res.end(`<html><script>window.open('${serverUrl}/second-window-open', 'first child');</script></html>`);
```

Comment:
Does this have to open another child window in `/second-window-open` to repro the issue, wouldn't it repro if you used `res.end(<html><script>window.open('${serverUrl}/redirect-cross-site', 'first child');</script></html>);` ?

Reply:
I tried that and it didn't work (i.e. problem doesn't reproduce). It should reproduce if there would be redirection from http to https (first reproduction and the most probable in normal usage was on http://www.google.com which redirects to https://www.google.com) but to achieve it in the test it would be necessary to start https server with valid certificate next to http server.

Reply:
the protocol doesn't matter here `http://www.google.com/` and `https://www.google.com/` are different origins for chromium, it is required for a cross origin redirect to repro this issue. But weird, maybe the testing needs a network delay. Anyways the current test is good, wanted to see if we can simplify it. Thanks!